airbyte-ci/ci/ci_credentials/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+[tool.poetry]
+name = "ci_credentials"
+version = "1.1.0"
+description = "CLI tooling to read and manage GSM secrets"
+authors = ["Airbyte <contact@airbyte.io>"]
+readme = "README.md"
+packages = [{ include = "ci_credentials" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+requests = "^2.28.2"
+click = "^8.1.3"
+pyyaml = "^6.0"
+ci_common_utils = { path = "../ci_common_utils", develop = true }
```

Comment:
If we go ahead with this we will need to update the local dependency resolution right?